pkg/dc2deployment/dc2deployment.go

```diff
@@ -96,13 +87,63 @@ func Mutate(pluginName string, dc dcAPI.DeploymentConfig, log *logrus.Logger) (d
 			deploy.Spec.Selector.MatchLabels[index] = element
 		}
 	}
-	log.Debugf("[%s] d.Selector: %#v", pluginName, deploy.Spec.Selector)
+	log.Debugf("[%s] Strategy: %#v", pluginName, dc.Spec.Strategy)
+
+	// openshift supoorts Strategy.Type={"Recreate","Custom","Rolling"}
+	// K8s native supports k8s.Strategy.Type={"Recreate","RollingUpdate"}
+	// Custom strategy.type is not supported in K8s native, hence defaulting it to RollingUpdate
+	if dc.Spec.Strategy.Type != "" {
+		if dc.Spec.Strategy.Type == "Rolling" || dc.Spec.Strategy.Type == "Custom" {
+			deploy.Spec.Strategy.Type = deployAPI.DeploymentStrategyType("RollingUpdate")
+		} else {
```

Comment:
Let's make a comment here that this is expected to be "Recreate"

Reply:
Added comment for the Recreate Strategy.Type.